internal/pkg/template/templates/workloads/partials/cf/nlb.yml

```
@@ -10,58 +10,66 @@ PublicNetworkLoadBalancer:
         - Fn::ImportValue:
             !Sub '${AppName}-${EnvName}-PublicSubnets'
     Type: network
-
+{{- range $i, $listener := .NLB.Listener }}
+{{- if eq $i 0 }}
 NLBListener:
+{{- else }}
+NLBListener{{$i}}:
```

we can do the same thing here now 👍 
```suggestion
NLBListener{{- if ne $i 0 }}{{$i +2}}{{- end}}:
```